broScripts/parse_pcap.py

```diff
@@ -54,6 +54,31 @@ def summary_stats():
 def is_person_empty(field):
     return field == '' or field == '-' or field == '<>' or field == '(empty)' or field == 'undisclosed'
 
+def parseLine(line):
+    lstOfTups = []
+
+    while True:
+        if "('" not in line or "')" not in line or "','" not in line:
+            return lstOfTups
+        startParen = line.index("('")
+        endParen = line.index("')")+1
+        comma = line.index("','") + 1
```

Comment:
Will we ever have to worry about things that don't match this exact format? (i.e. `', '` instead of `','`). Would it be better to use some regex to capture the potential whitespace differences?